### ISSUE TITLE
[BUG] TypeError: object of type 'NoneType' has no len()

### ISSUE THREAD

**Issue by ghost** [interactions-py/interactions.py#649]
### Describe the bug.

Attempting to use a subcommand results in a "TypeError"

### List the steps.

```
@inter.command(
    name="replace",
    description="チャットの置換表の管理を行います",
    options=[
        interactions.Option(
            name="add",
            description="追加",
            type=interactions.OptionType.SUB_COMMAND,
            options=[
                interactions.Option(
                    name="word",
                    description="削除するワードを指定してください",
                    type=interactions.OptionType.STRING,
                    required=True,
                ),
                interactions.Option(
                    name="after",
                    description="置換先の文字列",
                    type=interactions.OptionType.STRING,
                    required=True,
                ),
            ],
        ),
        interactions.Option(
            name="remove",
            description="削除",
            type=interactions.OptionType.SUB_COMMAND,
            options=[
                interactions.Option(
                    name="word",
                    description="削除するワードを指定してください",
                    type=interactions.OptionType.STRING,
                    required=True,
                ),
            ],
        ),
        interactions.Option(
            name="clear",
            description="置換表のリセットを行います",
            type=interactions.OptionType.SUB_COMMAND,
        ),
    ],
)
async def replace_(ctx, sub_command: str, mode, word, after):
```

### What you expected.

Using commands
I am migrating from "discard_components" + "discard-slash" + "discard.py"
Moving to "interactions.py" + "discard.py" with the goal of using modal, etc.

### What you saw.

Instead, I received this traceback error given from my Python terminal:
```
Traceback (most recent call last):
  File "D:\Desktop\Projects\Discord\meru\meru_bot_v2.py", line 340, in <module>
    async def replace_(ctx, sub_command: str, mode, word, after):
  File "C:\Users\ikki1\AppData\Local\Programs\Python\Python39\lib\site-packages\interactions\client.py", line 593, in decorator
    self.__check_command(command=ApplicationCommand(**commands[0]), coro=coro)
  File "C:\Users\ikki1\AppData\Local\Programs\Python\Python39\lib\site-packages\interactions\client.py", line 522, in __check_command
    __check_sub_command(_option)
  File "C:\Users\ikki1\AppData\Local\Programs\Python\Python39\lib\site-packages\interactions\client.py", line 403, in __check_sub_command
    if len(_sub_command.options) > 25:
TypeError: object of type 'NoneType' has no len()
```

### What version of the library did you use?

stable

### Version specification

4.1.0

### Code of Conduct

- [X] I agree to follow the contribution requirements.

**Comment by kylebeee** [comment]
I had the same issue. Definitely need clearer error logging for this particular problem but your code should work if you add options=[] to your last sub command. I was also getting hung up because even after i figured that out i was getting cryptic errors and it turned out that i needed to remove required=True

**Comment by sillyElvy** [comment]
Hello! Iirc this issue has already been fixed, and the fix should be available on 4.1.1-beta.1 if you wish to check it out

**Comment by sillyElvy** [comment]
Closing since lack of updates
